backend/src/models/User.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   id: { type: 'string', primary: true, default: uuid }, // TODO: should be type: 'uuid' but simplified for our tests
   actorId: { type: 'string', allow: [null] },
   name: { type: 'string', disallow: [null], min: 3 },
-  slug: 'string',
+  slug: { type: 'string', regex: /^[a-z0-9_-]+$/, lowercase: true },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-23T06:25:37Z" title="Monday, September 23rd 2019, 8:25:37 am +02:00">Sep 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -4,7 +4,7 @@ module.exports = {
   id: { type: 'string', primary: true, default: uuid }, // TODO: should be type: 'uuid' but simplified for our tests
   actorId: { type: 'string', allow: [null] },
   name: { type: 'string', disallow: [null], min: 3 },
-  slug: 'string',
+  slug: { type: 'string', regex: /^[a-z0-9_-]+$/, lowercase: true },
```


we could maybe add `unique: true` here as well, but it can be a refactor, for sure